reqs/management/commands/import_reqs.py

```diff
@@ -42,6 +42,19 @@ def convert_date(string):
             raise ValueError("Not a date: {0}".format(string))
 
 
+def find_key(options, row, highlander=True):
```

Comment:
Looks like highlander's never used and I'm not sure what it's supposed to do -- can we drop it?